sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 ## Release History
-
-### 1.3.0-beta.1 (Unreleased)
-
-#### Features Added
-
-#### Breaking Changes
-
-#### Bugs Fixed
-
-#### Other Changes
+### 1.3.0 (2022-08-15)
+### Other Changes
```

Comment:
We can do something like what ACS Identity did [here](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/communication/azure-communication-identity/CHANGELOG.md#features-added-1). Feel free to modify it as you see fit.
```suggestion
### New features
- Added `String getRawId()` and `static CommunicationIdentifier fromRawId(String rawId)` to `CommunicationIdentifier` to translate between a `CommunicationIdentifier` and its underlying canonical `rawId` representation. Developers can now use the `rawId` as an encoded format for identifiers to store in their databases or as stable keys in general.

### Other Changes
```